protos/dcs/tts/v0/tts.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+package dcs.tts.v0;
+import "dcs/common/v0/common.proto";
+option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Tts";
+option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/tts";
+
+service TtsService {
+  rpc Transmit(TransmitRequest) returns (TransmitResponse) {}
+}
+
+message TransmitRequest {
+  // The text that is synthesized to speech and transmitted to SRS.
+  string text = 1;
+
+  // The radio frequency in Hz the transmission is send to. Example: 251000000
+  // for 251.00MHz.
+  uint64 frequency = 2;
+
+  // Name of the SRS client. Defaults to "DCS-gRPC".
+  optional string name = 3;
+
+  // The origin of the transmission. Relevant if the SRS server has "Line of
+  // Sight" and/or "Distance Limit" enabled.
+  dcs.common.v0.Position position = 4;
+
+  // The coalition of the transmission. Relevant if the SRS server has "Secure
+  // Coalition Radios" enabled. Only Blue and Red are supported, all other
+  // values will fallback to Blue.
+  dcs.common.v0.Coalition coalition = 5;
+
+  // The socket (hostname:port) of the SRS server to connect to. Defaults to
+  // 127.0.0.1:5002 if not specified.
+  optional string srs_addr = 6;
+
+  // Whether to keep the request open until the whole transmission was sent. If
+  // enabled, you can send the next transmission after you've received the
+  // response for the previous one and be sure that they don't overlap (talk
+  // over each other). If disabled, you'll receive a response right away (kind
+  // of fire and forget).
+  bool wait = 7;
```

Comment:
Would `async` be a better name for this? Or `blocking`? Both of these are industry standard terms which might make it more obvious.

Reply:
I derived `wait` from the `--no-wait` flag I often see in CLIs. But I am totally up for another term to make it easier to understand.

I was also considering `blocking`, but avoided it to not confuse it with blocking the DCS mission (doesn't really make sense that this would block DCS, but it came to my mind).

`async` would work for me.

Another idea: get rid of the field altogether and instead return the time in milliseconds it will roughly take to _speak_ the transmission. The client could then sleep that amount. This would have the benefit of avoiding running into request timeouts (in case there are any due to proxies or whatever).

Reply:
I think returning the time is a good idea even if we have async option but do all the providers support that?

Reply:
I don't get the time from the providers (actually I haven't checked, I'd have just done it as follows), I'd have to make the following changes:
- It always executes the speech synthesizes and opus encoding before the response is send to the client.
- Which is necessary so that I can then estimate the duration based on the amount of opus audio frames.

It looks as follows right now (in the `async` case):

```
Request -> Open Connection to SRS -> Response -> Speech Synthezis -> OPUS encoding -> SRS Transmission
```

To return the Duration, I'd change it to:

```
Request -> Open Connection to SRS -> Speech Synthezis -> OPUS encoding -> Response -> SRS Transmission
```

Reply:
Hmm. What would the performance impact of that be? I am fine with async being an option but false by default.

Reply:
The TTS will definitely take a bit longer, as it is waiting for the round-trip to the TTS provider. However, this might be a good thing because it allows to return a possible error from the TTS provider even if `async` is enabled.

1074ee06f64f0fe7b27ca32235f23ee65a693696 renames the field to `async`, and adds the `duration_ms` to the response